specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2021-10-01/operations.json

```diff
@@ -40,7 +40,7 @@
         "description": "Lists all operations available Azure Security Insights Resource Provider.",
         "parameters": [
           {
-            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
           }
         ],
         "produces": [
```

Comment:
According to Linter error, Looks in L56, there is one more CloudError need to change to common definition

Reply:
Thank you